package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
You need to install react, you can go ["here"](https://create-react-app.dev/docs/getting-started/) to get started. It will create a new folder at the root level of the lab. Before you move any file/folders remove your package.json, package-lock.json, and .gitignore,(also node_modules if you have it), because when you install react it comes with all of that(which is inside the new folder). Then move your components folder inside the the new folder  to inside the src folder(i.e. : newfoldername/src).  

In the new folder, the one created by the react install, there is a readme that tell the different npm commands you can use; rename it to oldReadme.md or something similar so you have a reference. Then remove App.js and replace it with your's. Then move all the folders out of the new folder to the root level. The new folder should be empty.  Before you run `npm start` you'll need to run `npm i node-sass` to install scss. After that run `npm start` and it should open a new window/tab to localhost:3000

If your having some trouble slack me!